explorer/src/__tests__/realtime.test.ts

```diff
@@ -182,4 +182,24 @@ describe('realtime', () => {
       },
     )
   })
+
+  it('rejects multiple connections from single node', async done => {
+    expect.assertions(4)
+
+    let ws1: WebSocket
+    let ws2: WebSocket
+
+    const onCloseCallback = (event: WebSocket.CloseEvent) => {
+      expect(ws1.readyState).toBe(3) // connection closed
+      expect(ws2.readyState).toBe(1) // connection open
+      expect(event.code).toBe(1000)
```

Comment:
How do you feel about defining and using a custom constant for this? e.g. `const NORMAL_CLOSE: number = 1000`